CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ IMPROVEMENTS:
 * Use `consul-k8s` subcommand to perform `tls-init` job. This allows for server certificates to get rotated on subsequent runs.
   Consul servers have to be restarted in order for them to update their server certificates [[GH-749](https://github.com/hashicorp/consul-helm/pull/721)]
 
+BUG FIXES:
+* Consul servers no longer call `consul leave` command when restarted or deleted. [[GH-764](https://github.com/hashicorp/consul-helm/pull/764)]
```

Comment:
🚗 could we also say that this is okay because we don't want them to leave anyway since that reduces the quorum size.

Reply:
Ah, good point, will add! Always love the 🚗 comments 😄

Reply:
Updated!